### ISSUE TITLE
speaker diarization pipeline

### ISSUE THREAD

**Issue by Wulin824** [pyannote/pyannote-audio#672]
I am using the pre-training models emb-ami, sad-ami and scd-ami to train the speaker dialysis pipeline, which is my own data set. Because this training took too long, I stopped it halfway and generated several files. I wonder if this training can continue after the last training?
![微信图片_20210520095016](https://user-images.githubusercontent.com/56509789/118906678-ce9f4a00-b950-11eb-9536-3d38bc268d0b.png)

**Comment by hbredin** [comment]
Yes, you can resume training simply by calling `pyannote-pipeline train` again from the same directory - it will pick up from where it stopped.
